chrome/locale/en-US/zotero/zotero.dtd

```diff
@@ -96,6 +96,7 @@
 <!ENTITY zotero.items.menu.mergeItems					"Merge Items…">
 <!ENTITY zotero.items.menu.unrecognize                "Undo Retrieve Metadata">
 <!ENTITY zotero.items.menu.reportMetadata             "Report Incorrect Metadata">
+<!ENTITY zotero.items.menu.updateMetadata             "Update Metadata">
```

Comment:
We use "metadata" in a few contexts, but it's probably a mistake — few people know what that means.

Also, doesn't have to be with this commit, but won't we want this function to retrieve PDFs if they're now available (say, because we resolved a DOI, or the PDF is now available on the page that we're translating anyway)? We could leave that separate, but I think it'd make sense that you'd basically update an item and get everything that's now available for it, both metadata and PDF.

I'd say "Update Item", but we're [removing the extraneous "Item" references](https://github.com/zotero/zotero/pull/1535) from the context menu. "Update Details" doesn't include the PDF.  "Update" alone might work, though it's a little vague.

I guess just "Update" for now, and we can see if we come up with something better? If it's just "Update", do we need multiple plural forms (e.g., for Lithuanian) to reflect the number of items selected (or at least singular vs. plural)?

(And the key can be `updateItem` — if we incorporate PDF handling, I think that becomes the internal name of the feature.)

Reply:
That would be very convenient to have just one context menu option to just update/retrieve everything, but for the beginning it's better to make sure that metadata updating is working good enough.

But there are a few things to discuss.
So if an item doesn't have any PDF, we can just retrieve them, which is simple, but if there already are some PDFs we need a way to update them, which can be complicated. What if user added PDFs manually?

Then another thing to discuss is how are we going to implement undo, especially if we'll combine metadata update with PDF retrieval.

Reply:
The last commit avoids unnecessary localizations for metadata update, only `updateMetadata.title` is left which is needed for progress window title. As I said, I agree that we should combine metadata update with PDF retrieval but that should happen after the metadata updating proves itself.

Reply:
Perhaps "Update Details" and "Update Details and Find Available PDF" if no attachment is present?

Reply:
Well, maybe, but in future we would like to update attachments too (if it was a preprint i.e.). Although that can be complicated to achieve in practice.